marshmallow_dataclass/__init__.py

```diff
@@ -74,14 +72,6 @@ class User:
 MAX_CLASS_SCHEMA_CACHE_SIZE = 1024
 
 
-class BaseSchema(marshmallow.Schema):
-    TYPE_MAPPING = {
-        SerializableAttribute: marshmallow.fields.Mapping,
-        SerializableAttributeTag: marshmallow.fields.Mapping,
-        TypedDict: marshmallow.fields.Mapping,
-    }
-
-
```

Comment:
there would be a circular dependency if blib/specs needed marshmallow_dataclass, but marshmalow_dataclass imported these types